sound/soc/sof/ipc4-topology.c

```diff
@@ -1924,6 +1924,36 @@ static int sof_ipc4_dai_get_clk(struct snd_sof_dev *sdev, struct snd_sof_dai *da
 	return -EINVAL;
 }
 
+static int sof_ipc4_tear_down_all_pipelines(struct snd_sof_dev *sdev, bool verify)
```

Comment:
@ujfalusi I think this change is good also for paused streams.

Reply:
is it? For paused streams we don't get a TRIGGER_SUSPEND, do we?

Reply:
Looks that way, yes. And because of that this patch makes that use case also working.